tools/esdlcmd/esdlcmd_shell.cpp

```diff
@@ -180,7 +180,7 @@ bool EsdlCMDShell::parseCommandLineOptions(ArgvIterator &iter)
 void EsdlCMDShell::usage()
 {
     fprintf(stdout,"\nUsage:\n"
-        "    esdl [--version] <command> [<args>]\n\n"
```

Comment:
Sounds reasonable, I'll replace --version with --interface-version and -iv, then replace --build with --verison.